Doc/reference/datamodel.rst

```diff
@@ -1069,8 +1069,9 @@ Internal types
       :attr:`f_code` is the code object being executed in this frame; :attr:`f_locals`
       is the dictionary used to look up local variables; :attr:`f_globals` is used for
       global variables; :attr:`f_builtins` is used for built-in (intrinsic) names;
-      :attr:`f_lasti` gives the precise instruction (this is an index into the
-      bytecode string of the code object).
+      :attr:`f_lasti` gives the precise instruction (it represents a wordcode index, which
```

Comment:
This is incorrect. f_lasti is the index in the bytecode. No need to multiply by two.

Reply:
Fixed in https://github.com/python/cpython/pull/28208